scripts/unmount.sh

```diff
@@ -11,5 +11,19 @@ if [ "$(readlink /proc/self/ns/mnt)" != "$(readlink /proc/1/ns/mnt)" ]; then
   exit 1
 fi
 
-umount -v /mnt/runtime/write/emulated/0/the_binding
-umount -v /mnt/my_drive
+android_version=$(getprop ro.build.version.release)
+
+if [ $android_version -gt 10 ]; then
+  # for Android 11+
+  internal_binding_dir="/mnt/pass_through/0/emulated/0/the_binding"
+else
+  # for Android 10 and below
+  internal_binding_dir="/mnt/runtime/write/emulated/0/the_binding"
+fi
+
+[[ $(mount | grep -w "$internal_binding_dir") != "" ]] && \
```

Comment:
i don't think checking `mount` is necessary - it is totally okay for `umount` to fail

Reply:
same for below

Reply:
Though I do not really agree in terms of design :) but its okay anyway

Reply:
my goal is to keep the scripts as simple as possible - i think the proposed mount check increases complexity with the same outcome.

also i think the better way to check for existing mount would be with `stat` rather than by grepping through the list of all mounts